fe/fe-core/src/main/java/org/apache/doris/planner/RuntimeFilterGenerator.java

```diff
@@ -181,7 +203,7 @@ public static void generateRuntimeFilters(Analyzer analyzer, PlanNode plan) {
      * In the bottom-up traversal of the plan tree, the filters are assigned to destination
      * (scan) nodes. Filters that cannot be assigned to a scan node are discarded.
      */
-    private void generateFilters(PlanNode root) {
+    private void generateFilters(PlanNode root, HashSet<TupleId> tupleHasConjuncts) {
```

Comment:
Make `tupleHasConjuncts` a member variable of `RuntimeFilterGenerator` to avoid passing it frequently.

Reply:
Done